### ISSUE TITLE
Firmware Updater

### ISSUE THREAD

**Issue by yoshiask** [yoshiask/ZuneModdingHelper#4]
Add the `zuneupdate.com/resources` mod that allows the updating of device firmware.
Needs to point `resources.zune.net` (and potentially `resources.zunes.tk`) to https://www.zuneupdate.com/resources/zune-hosts-file.zip (`66.115.173.227`)

**Comment by andrew2025nmwa-afk** [comment]
If you are on windows you can open the hosts file as a admin then add the server IP then press tab then do the server resources website then save it then It should work.

It worked for me and I was able to restore my zune and get it updated.

**Comment by yoshiask** [comment]
Yes, that works but this point of this tool is to not require knowledge about how Zune or the mods work.
